neuralop/models/codano.py

```diff
@@ -0,0 +1,371 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import torch
+import numpy as np
+from ..layers.channel_mlp import ChannelMLP
+from ..layers.spectral_convolution import SpectralConv
+from ..layers.skip_connections import skip_connection
+from ..layers.padding import DomainPadding
+from ..layers.coda_blocks import CODABlocks
+from ..layers.resample import resample
+from ..layers.embeddings import GridEmbedding2D, GridEmbeddingND
+
+class CODANO(nn.Module):
+    """
+    Parameters
+    ---
+    output_variable_codimension : int
```

Comment:
If you provide default values for arguments, they should be noted in the docstring

Reply:
added